### PR TITLE
Improve trivia plugin

### PR DESCRIPTION
**Pull request by Bryan-0**
- Add a reply if a "Trivia" or "Question Workshop"  command is used in other room.

### CODE REVIEW

**Comment by Bryan-0** [comment]
``[15:09:31] Error in plugin 'gulp-jshint'``
``JSHint failed for: /home/travis/build/Zarel/Pokemon-Showdown/chat-plugins/trivia.js`` 
Huh..

File: chat-plugins/trivia.js [hunk 1]
```diff
@@ -623,7 +627,9 @@ var commands = {
 	// question database modifying commands
 	submit: 'add',
 	add: function (target, room, user, connection, cmd) {
-		if (room.id !== 'questionworkshop' || (cmd === 'add' && !this.can('mute', null, room)) || !target) return false;
+		if (room.id !== 'questionworkshop') return this.sendReply('This command can only be used in Question Workshop.');
+		if (cmd === 'add' && !this.can('mute', null, room))
```
Comment (kotarou3) [inline chat-plugins/trivia.js]: ?

**Comment by Zarel** [comment]
So, are you planning on fixing these errors?

**Comment by Bryan-0** [comment]
I'm going to squash it.

**Comment by Bryan-0** [comment]
Updated. @Zarel

**Pull request merged**